specification/e-referrals-service-api.yaml

```diff
@@ -391,6 +400,80 @@ paths:
           $ref: 'components/schemas/responses/NotFound.yaml'
         '500':
           $ref: 'components/schemas/responses/InternalServerError.yaml'
+  /STU3/v1/Slot:
+    get:
+      description: |
+        ## Overview
+        Use this endpoint to retrieve available appointment slots for a service.
+
+        ## Pre-requisites
+        In order to use this endpoint you must be an authenticated e-RS user and use one of the following e-RS roles:
+          - `REFERRING_CLINICIAN`
+          - `REFERRING_CLINICIAN_ADMIN`
+
+        You need to have identified a `service`,
+
+          - that is on the shortlist of a referral either
+            - via [A005 - Retrieve referral request](https://developer.nhs.uk/apis/e-Referrals/explore_endpoint_a005.html) to obtain the shortlist associated with an existing referral OR
+            - via [A011 - Create referral](#api-Default-a011CreateReferral) to associate a shortlist with a new referral
+          - via [A010 - Patient service search](#api-Default-a010PatientServiceSearch) to find services that meet search criteria
+
```

Comment:
Is the indentation correct here?

Reply:
yeah we wanted a 2 level of unordered list here, unfortunately the preview in UAT does not make those bullet points any different. I've already raised it with platform team there as according to the tool they're using for markup it should be displayed in a different style.